app/components/affiliate-unit.js

```diff
@@ -0,0 +1,6 @@
+import Component from '@ember/component';
+
+export default Component.extend(
+  // Intentionally empty, no extra logic needed
```

Comment:
Is this required if we don't need it ?

Reply:
Yes, an error occurs if the component file doesn't exist.

Reply:
:emberjs: